core/spec/lib/tasks/migrations/migrate_address_names_spec.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+path = Spree::Core::Engine.root.join('lib/tasks/migrations/migrate_address_names.rake')
+
+RSpec.describe 'solidus:migrations:migrate_address_names' do
+  around do |example|
+    self.use_transactional_tests = false
```

Comment:
Can you please explain somewhere why we do need to not use transactional tests and cleanup database tables manually for these specs?

Reply:
Actually, we don't need this. This is a leftover from the early stage of the development when the code also added the index on the column , which is not the case anymore. It can be safely removed 👍